index.yaml

```diff
@@ -6966,21 +6936,6 @@ entries:
     - assets/rancher-backup/rancher-backup-1.0.200.tgz
     version: 1.0.200
   rancher-backup-crd:
-  - annotations:
```

Comment:
@vardhaman22 why is this PR removing this `rancher-backup-crd` entry on `index.yaml`?